supertokens_python/recipe/session/interfaces.py

```diff
@@ -78,16 +89,45 @@ async def create_new_session(
     ) -> SessionContainer:
         pass
 
+    @abstractmethod
+    def get_global_claim_validators(
+        self,
+        user_id: str,
+        claim_validators_added_by_other_recipes: List[SessionClaimValidator],
+        user_context: Dict[str, Any],
+    ) -> MaybeAwaitable[List[SessionClaimValidator]]:
+        pass
+
     @abstractmethod
     async def get_session(
         self,
         request: BaseRequest,
         anti_csrf_check: Union[bool, None],
         session_required: bool,
+        override_global_claim_validators: Optional[
+            Callable[
+                [SessionContainer, List[SessionClaimValidator], Dict[str, Any]],
+                MaybeAwaitable[List[SessionClaimValidator]],
+            ]
+        ],
         user_context: Dict[str, Any],
     ) -> Union[SessionContainer, None]:
         pass
 
+    @abstractmethod
+    async def assert_claims(
+        self,
+        session: SessionContainer,
+        override_global_claim_validators: Optional[
+            Callable[
+                [SessionContainer, List[SessionClaimValidator], Dict[str, Any]],
+                MaybeAwaitable[List[SessionClaimValidator]],
+            ]
+        ],
+        user_context: Dict[str, Any],
+    ) -> None:
+        pass
+
```

Comment:
The `validateClaimsInJWTPayload` and `validateClaimsForSessionHandle` functions are missing (they were added more to the node PR recently)

Reply:
Added in the new PR 

https://github.com/supertokens/supertokens-python/pull/211